### ISSUE TITLE
7.A Practice Exam icon alt text update

### ISSUE THREAD

**Issue by AnnMarieDavis** [Section508Coordinators/DHSA-TT-200-A#261]
## Issue Type 
- [X] Other, specify...[replace this text with your issue type description]

## Description
On the Zip Chip pages of the Practice Exam, the logo/icons are not identified in a way that many agencies would accept as being clear alternative text. "Company logo" does not clearly indicate what company logo is being presented for the running man image. 
![2 B running man logo](https://user-images.githubusercontent.com/31414225/61475941-e8893380-a959-11e9-98f2-8fa2853acc47.png)

## Location
Practice Exam pages using ZipChip theme.
http://52.23.182.152/dhsv5/dhscourse/Lesson1/exams/web-practice/testPages/index.php?KeyID=1571

## Expected Behavior / Recommendation
In order to PASS this requirement, the logo alt text needs to be changed for this icon. Change the code on the PRACTICE EXAM to the text noted below for every **passing** instance of the running man icon:
- Zip Chip (running man) icon

Note: Failing examples should be very clear due to missing the alt text or inadequate text. (Do not use text that is currently being used for passing this Test ID as failing text.)

**Comment by Tonylayton1** [comment]
Updated documentation.

Please update the alt tag for the running man icon as above. should be: "Zip Chip (running man) icon"

Icon is present at the top and bottom of each page. the one on the bottom pf page 3 has an failure version with no alt, which should remain the same.

**Comment by mikebarklage** [comment]
Updated.

**Comment by Tonylayton1** [comment]
Verified on dev

**Comment by AnnMarieDavis** [comment]
I verified this as I was retaking the Practice Exam.  Thank you for this change.
